packages/create-analog/template-angular-v14/package.json

```diff
@@ -15,26 +15,26 @@
   },
   "dependencies": {
     "@analogjs/router": "latest",
-    "@angular/animations": "^14.0.0",
```

Comment:
Revert these changes. Its ok to let the package manager determine the latest version to be installed